scripts/fhap_build.R

```diff
@@ -1,24 +1,77 @@
 # build the FHAP digital form
 
 source('scripts/packages.R')
-source('scripts/functions.R')
 
-form_raw <- readxl::read_xlsx('data/FHAP_template.xlsx')
+#' name the project directory we are burning to
+dir_project <- 'bcfishpass_20230516'
 
-form_types <- readxl::read_xlsx('data/FHAP_template.xlsx',
-                                skip = 4)
+#' name the form using the date and time
+#' we should be able to name the form the same in the active project but the files can be versioned
+file_name <- paste0('form_fhap_', format(lubridate::now(), "%Y%m%d"))
 
 
-cols_to_add <- c('stream',
-                 'date',
-                 'utm_start',
-                 'utm_end',
-                 'surveyor_1',
-                 'surveyor_2')
+#' define your utm zone.  This can cause errors if you use the form in more than
+#' one zone!!!!! beware
+utm_zone <- 11
 
-form_col_names <- form_raw %>%
-  wkb_col_names() %>%
-  stringr::str_replace_all(., 'd_s_utm_', '')
+#' import the fhap submission template (needs to be in the data directory)
+#' because we want to keep the backup file clean for the value maps and because
+#' we are not worried about version controlling this data we turn the `backup` function to `FALSE`
 
-# this will make a tibble but maybe we make one with populated options and let it guess
-form_col_names %>% purrr::map_dfc(setNames, object = list(character()))
+form_raw <- readxl::read_xlsx('data/FHAP_template.xlsx', skip = 1)
+
+# see the names of the columns
+names(form_raw)
+
+fhap <- form_raw %>%
+  janitor::clean_names() %>%
+  # add some columns of our own
+  dplyr::mutate(stream = NA_character_,
```

Comment:
there are two pages to the spreadsheet and the second one has field we want

Reply:
note how we bring in all the sheets for fpr::fpr_import_hab_con(). I wonder if  ` fpr::fpr_import_hab_con()` would work to read this in?

Reply:
I think that function should work ya. Because the path parameter takes a file that can be anything.